test/jdk/javax/swing/text/html/StyleSheet/bug8257665.java

```diff
@@ -0,0 +1,116 @@
+/*
+ * <Copyright>
+ */
+
+import java.util.concurrent.atomic.AtomicReference;
+import java.util.function.Consumer;
+import javax.swing.JEditorPane;
+import javax.swing.SwingUtilities;
+import javax.swing.text.GlyphView;
+import javax.swing.text.View;
+
+/*
+ * @test
+ * @key headless
+ * @bug 8257665
+ * @summary  Tests inherited font-size with parent percentage specification.
+ * @run main bug8257665
```

Comment:
We normally dont use bugid as name for the test. You probably can name it TestWrongCSSFontSize.java

Reply:
Renamed.  I had mimicked the existing `bug4936917.java`, initially.  I also had the `@bug` number wrong.